examples/views/dropdown.py

```diff
@@ -37,4 +38,5 @@ async def select(ctx: commands.Context):
     # Sending the message with the view
     await ctx.send('Pick your favourite color:', view = view)
 
-bot.run(TOKEN)
+import os
+bot.run(os.getenv('TOKEN'))
```

Comment:
```suggestion
bot.run('token')
```
it would be good to stay consistent with the rest of the examples

Reply:
Oops, sorry about that. Forgot to remove that part after testing.